camel/retrievers/bm25_retriever.py

```diff
@@ -54,6 +55,7 @@ def __init__(self) -> None:
         self.bm25: BM25Okapi = None
         self.content_input_path: str = ""
         self.chunks: List[Any] = []
```

Comment:
if using unstructured io the type of chunk would be `element`, if user do chunk by themselves it could be `str`, I think it's not worth to import `element` from third party library just for a type hint here

Reply:
Ok.